src/Xamarin.Android.Build.Tasks/Tasks/AndroidSignPackage.cs

```diff
@@ -42,7 +44,7 @@ protected override string GenerateCommandLineCommands ()
 			cmd.AppendSwitchIfNotNull ("-storepass ", StorePass);
 			cmd.AppendSwitchIfNotNull ("-keypass ", KeyPass);
 			cmd.AppendSwitchIfNotNull ("-digestalg ", "SHA1");
-			cmd.AppendSwitchIfNotNull ("-sigalg ", "md5withRSA");
+			cmd.AppendSwitchIfNotNull ("-sigalg ", string.IsNullorWitespace(SigningAlgorithm) ? "md5withRSA" :SigningAlgorithm);
```

Comment:
@fekberg we have a typo here

`IsNullorWitespace` vs `IsNullOrWitespace`

Reply:
Oh my, sorry about that!